frontend/public/components/modals/configure-update-strategy-modal.tsx

```diff
@@ -20,12 +23,14 @@ export const getNumberOrPercent = (value) => {
 };
 
 export const ConfigureUpdateStrategy: React.FC<ConfigureUpdateStrategyProps> = (props) => {
+  const { hideDescription = false } = props;
   return (
     <>
-      <div className="co-m-form-row">
-        <p>How should the pods be replaced when a new revision is created?</p>
-      </div>
-
+      {!hideDescription && (
```

Comment:
change prop to `showDescription` for clarity so you don't need the NOT and you're consistent with `showLabel`?